docs/source/usecases/track_data_from_webpage.rst.in

```diff
@@ -0,0 +1,185 @@
+   fair_networks.txt  figure_5_12.m  mean_func.nii.gz
+   figure_2_12.R	   figure_5_3.m   zstat1_0mm.nii.gz
+   figure_2_14.m	   figure_8_11.R  zstat1_16mm.nii.gz
+   figure_2_3.m	   figure_8_2.m   zstat1_32mm.nii.gz
+   figure_3_8.m	   figure_9_1.R   zstat1_4mm.nii.gz
+   figure_3_9.m	   figure_9_2.R   zstat1_8mm.nii.gz
+   [1;36m~/demo[0m % #
+
+The webpage can be queried for potential updates at any time by
+re-running the 'crawl' command.
+
+.. code-block:: ansi-color
+
+   [1;36m~/demo[0m % datalad crawl
+   [[1;37mINFO   [0m] Loading pipeline specification from ./.datalad/crawl/crawl.cfg 
+   [[1;37mINFO   [0m] Creating a pipeline to crawl data files from http://www.fmri-data-analysis.org/code 
+   [[1;37mINFO   [0m] Not adding annex.largefiles=exclude=README* and exclude=LICENSE* to git annex calls because already defined to be (not(mimetype=text/*)) 
+   [[1;37mINFO   [0m] Running pipeline [<function switch_branch at 0x7f47abaf3488>, [[<datalad.crawler.nodes.crawl_url.crawl_url object at 0x7f479a6fcd50>, a_href_match(query='.*d=1$'), <function fix_url at 0x7f47b023acf8>, <datalad.crawler.nodes.annex.Annexificator object at 0x7f479a6dd810>]], <function switch_branch at 0x7f479a6e3de8>, [<function merge_branch at 0x7f479a6e3050>, [find_files(dirs=False, fail_if_none=True, regex='\\.(zip|tgz|tar(\\..+)?)$', topdir='.'), <function _add_archive_content at 0x7f479a6e3e60>]], <function switch_branch at 0x7f479a6e3ed8>, <function merge_branch at 0x7f479a6e3f50>, <function _finalize at 0x7f479a706050>] 
+   [[1;37mINFO   [0m] Found branch non-dirty -- nothing was committed 
+   [[1;37mINFO   [0m] Checking out an existing branch incoming 
+   [[1;37mINFO   [0m] Fetching 'http://www.fmri-data-analysis.org/code' 
+                                                                                   [[1;37mINFO   [0m] Found branch non-dirty -- nothing was committed 
+   [[1;37mINFO   [0m] Checking out an existing branch incoming-processed 
+   [[1;37mINFO   [0m] Found branch non-dirty -- nothing was committed 
+   [[1;37mINFO   [0m] Checking out an existing branch master 
+   [[1;37mINFO   [0m] Finished running pipeline: URLs processed: 16,  Files processed: 16, skipped: 16 
+   [[1;37mINFO   [0m] Total stats: URLs processed: 16,  Files processed: 16, skipped: 16,  Datasets crawled: 1 
+
+
+Files can be added, or removed from this dataset without impairing
+the ability to get updates from the webpage. DataLad keeps the
+necessary information in dedicated Git branches.
+
+.. code-block:: ansi-color
+
+   [1;36m~/demo[0m % git branch
+     git-annex[m
+     incoming[m
+     incoming-processed[m
+   * [32mmaster[m
+
+
+ _____________________________________
+/ Demo was using datalad 0.9.2.dev1.  \
+\ Discover more at http://datalad.org /
+ -------------------------------------
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
+
```

Comment:
I see no point in having lots of cows with version info pop up in the docs. In your example it might be tolerable, but some casts are used as part of other docs.

Reply:
I was just confused how come they are not appearing in the rendering at all?  just for my own sake of understanding the process ;)